bleak/backends/bluezdbus/client.py

```diff
@@ -454,32 +470,34 @@ async def _pair(self) -> Message:
         assert self._bus is not None
         assert self._device_path is not None
 
-        # REVIST: This leaves "Trusted" property set if we
-        # fail later. Probably not a big deal since we were
-        # going to trust it anyway.
-        # Trusted means device is authorized
-        reply = await self._bus.call(
-            Message(
-                destination=defs.BLUEZ_SERVICE,
-                path=self._device_path,
-                interface=defs.PROPERTIES_INTERFACE,
-                member="Set",
-                signature="ssv",
-                body=[defs.DEVICE_INTERFACE, "Trusted", Variant("b", True)],
+        if callbacks is None:
```

Comment:
Why does this depend on callbacks?

Maybe we should move this after the call to "Pair"?